plutus-benchmark/coop/test/9.6/mustBurnOwnSingleton.golden.eval

```diff
@@ -1,6 +1,3 @@
-CPU:               114_113_035
-Memory:                579_805
-AST Size:                1_274
-Flat Size:               3_833
-
-(con unit ())
\ No newline at end of file
+An error has occurred:
+The machine terminated because of an error, either from a built-in function or from an explicit use of 'error'.
+Caused by: error
```

Comment:
Looks like you've introduced a bug somewhere

Reply:
yep, can't really figure this out either. Only thing changed is using builtin list utilities instead of SOP one